docs/reference/sql/apis/sql-search-api.asciidoc

```diff
@@ -54,8 +54,13 @@ precedence.
 [[sql-search-api-request-body]]
 ==== {api-request-body-title}
 
+`catalog`::
+(Optional, string) Default catalog (cluster) for queries. If unspecified, the
+queries execute on the data in the local cluster only. See
+<<modules-cross-cluster-search, cross cluster search>>.
+
 `columnar`::
-(Optional, Boolean) If `true`, returns results in a columnar format. Defaults to
+(Optional, boolean) If `true`, returns results in a columnar format. Defaults to
```

Comment:
I think this is supposed to be capitalized unless we're referring to something like the `bool` query.

Reply:
The other types (`string`, `integer`, but also `object`, `time values`, `array`)  are all lowercase. Shouldn't we capitalize them all or none?

Reply:
I see. Thanks, reverted.